argumentsExt/Resolver/ArgumentsResolver.php

```diff
@@ -1,42 +1,40 @@
 <?php
 
 /*
- * This file is part of the ScenarioStateBehatExtension project.
+ * This file is part of the StepArgumentInjectorBehatExtension project.
  *
  * (c) Rodrigue Villetard <rodrigue.villetard@gmail.com>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
 
-namespace Gorghoa\ScenarioStateBehatExtension\Resolver;
+namespace Gorghoa\StepArgumentInjectorBehatExtension\Resolver;
 
 use Doctrine\Common\Annotations\Reader;
-use Gorghoa\ScenarioStateBehatExtension\Annotation\ScenarioStateArgument;
-use Gorghoa\ScenarioStateBehatExtension\Context\Initializer\ScenarioStateInitializer;
+use Gorghoa\StepArgumentInjectorBehatExtension\Annotation\StepInjectorArgument;
 
 /**
  * @author Vincent Chalamon <vincent@les-tilleuls.coop>
  */
 class ArgumentsResolver
 {
     /**
-     * @var ScenarioStateInitializer
+     * @var Reader
      */
-    private $store;
+    private $reader;
 
     /**
-     * @var Reader
+     * @var array
      */
-    private $reader;
+    private $hookers;
```

Comment:
But be at ease, i see the politically incorrect and discomfort  that this naming carry.